services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_scheduler.py

```diff
@@ -34,12 +62,14 @@ class ObservationItem(BaseModel):
 async def update_service_observation(
     node_uuid: NodeID,
     observation_item: ObservationItem,
-    dynamic_sidecars_scheduler: DynamicSidecarsScheduler = Depends(
-        get_dynamic_sidecar_scheduler
-    ),
+    dynamic_sidecars_scheduler: Annotated[
+        DynamicSidecarsScheduler, Depends(get_dynamic_sidecar_scheduler)
+    ],
 ) -> NoContentResponse:
-    if dynamic_sidecars_scheduler.toggle_observation(
-        node_uuid, observation_item.is_disabled
+    if _retry_toggle_observation(
```

Comment:
also wondering here a bit if that makes sense to rename this. but that is a very low level thought...

Reply:
what pattern would you use here for the renaming?

Reply:
I don't really see why it's called retry. but as I said. it's a minor thing.